tools/renderer_html.py

```
@@ -0,0 +1,620 @@
+            "-filter_complex", filter_complex,
+            "-map", "[v]",
+            "-r", str(fps),
+            "-c:v", "libx264",
+            "-crf", str(crf),
+            "-preset", "medium",
+            "-y",  # Overwrite output
+            str(out_path)
+        ]
+        
+        try:
+            subprocess.run(cmd, capture_output=True, text=True, check=True)
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)